helper/testhelpers/testhelpers.go

```diff
@@ -612,6 +632,38 @@ func WaitForNCoresSealed(t testing.T, cluster *vault.TestCluster, n int) {
 	t.Fatalf("%d cores were not sealed", n)
 }
 
+func WaitForActiveNodeAndPerfStandbys(t testing.T, cluster *vault.TestCluster) {
+	t.Helper()
+	var standbys, actives int64
+	var wg sync.WaitGroup
+	deadline := time.Now().Add(15 * time.Second)
+	for _, c := range cluster.Cores {
+		wg.Add(1)
+		go func(client *api.Client) {
+			defer wg.Done()
+			for time.Now().Before(deadline) {
+				time.Sleep(250 * time.Millisecond)
+				leader, err := client.Sys().Leader()
+				if err != nil {
+					if strings.Contains(err.Error(), "Vault is sealed") {
+						continue
+					}
+					t.Fatal(err)
+				}
+				if leader.IsSelf {
+					atomic.AddInt64(&actives, 1)
+					return
+				}
+				if leader.PerfStandby && leader.PerfStandbyLastRemoteWAL > 0 {
+					atomic.AddInt64(&standbys, 1)
+					return
+				}
+			}
+		}(c.Client)
+	}
+	wg.Wait()
```

Comment:
Hmm, looks like the bit that actually tests standbys and actives is missing.  Will add.